src/commons/components/ceph-dashboard/ceph-dashboard.js

```diff
@@ -0,0 +1,269 @@
+            {
+                "iconClass":"fa fa-arrow-circle-o-down",
+                "count": downClusters,
+            },
+            {
+                "iconClass":"pficon  pficon-resources-almost-full",
+                "count": (clusterCountData.near_full ? clusterCountData.near_full : 0),
+            },
+            {
+                "iconClass":"pficon pficon-error-circle-o",
+                "count":(clusterCountData.critical_alerts ? clusterCountData.critical_alerts : 0),
+            },
+            {
+                "iconClass":"pficon pficon-warning-triangle-o",
+                "count":(clusterCountData.warning_alerts ? clusterCountData.warning_alerts  : 0),
+            },
+            {
+                "iconClass":"pficon pficon-flag",
+                "count": 0,
+                "text": " Cluster have host quorum"
+            }]
+        };
+    }
+
+    function clusterHost(clusterData){
+        var hostData = clusterData.hosts_count;
+        vm.hostStatus = {
+        "title":"Hosts",
+        "count": hostData.total,
+        "notifications":[
+        ]};
+
+        if(!hostData.down && !hostData.crit_alert_count && !hostData.warn_alert_count){
+            vm.hostStatus.notifications.push({
+                "iconClass":"pficon pficon-ok"
+            });
+        } else {
+            if(hostData.crit_alert_count){
+                vm.hostStatus.notifications.push({
+                    "iconClass":"pficon pficon-error-circle-o",
+                    "count": (hostData.crit_alert_count ? hostData.crit_alert_count : 0)
+                });
+            }
+            if(hostData.warn_alert_count){
+                vm.hostStatus.notifications.push({
+                    "iconClass":"pficon pficon-warning-triangle-o",
+                    "count": (hostData.warn_alert_count ? hostData.warn_alert_count : 0)
+                });
+            }
+            if(hostData.down){
+                vm.hostStatus.notifications.push({
+                    "iconClass":"fa fa-arrow-circle-o-down",
+                    "count": (hostData.down ? hostData.down : 0)
+                });
+            }
+        }
+    }
+
+    function clusterMon(){
+	    var monData = vm.cephCluster.sds_det.mon_counts;
+	    vm.monStatus = {
+	    "title":"Monitors",
+	    "count": monData.total,
+	    "notifications":[
+	    ]};
+
+	    if(!monData.outside_quorum){
+	        vm.monStatus.notifications.push({
+	            "iconClass":"pficon pficon-ok"
+	        })
+	    } else {
+	        if(monData.outside_quorum){
+	            vm.monStatus.notifications.push({
+	                "iconClass":"pficon pficon-flag",
+	                "count": (monData.outside_quorum ? monData.outside_quorum : 0)
+	            })
+	        }
+	    }
+	}
+
+	function clusterOsd(){
+        var osdData = vm.cephCluster.sds_det.osd_counts;
+        vm.osdStatus = {
+        "title":"OSDs",
+        "count": osdData.total,
+        "notifications":[
+        ]};
```

Comment:
@cloudbehl It can be written as "notifications": [] instead of closing "]" on next line.